src/main/java/no/sikt/nva/pubchannels/handler/fetch/publisher/FetchByIdAndYearResponse.java

```diff
@@ -0,0 +1,149 @@
+package no.sikt.nva.pubchannels.handler.fetch.publisher;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import no.sikt.nva.pubchannels.handler.ScientificValue;
+import no.sikt.nva.pubchannels.handler.fetch.ThirdPartyPublicationChannel;
+import no.sikt.nva.pubchannels.model.Contexts;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.paths.UriWrapper;
+
+import java.net.URI;
+import java.util.Objects;
+
+public class FetchByIdAndYearResponse {
+
+    private static final String TYPE_FIELD = "type";
```

Comment:
What about ISBN for publishers?

Reply:
We don\t get ISBN for publisher from dataporten. Are we missing something? [GET](https://kar-test.dataporten-api.no/swagger-ui.html#!/find45publisher/getFindKanalListeByAarPidUsingGET_1) [/findpublisher/{pid}/{year}](https://kar-test.dataporten-api.no/swagger-ui.html#!/find45publisher/getFindKanalListeByAarPidUsingGET_1)

Reply:
Not sure it's relevant as the data is limited and we don't use it